common/Cargo.toml

```diff
@@ -15,11 +15,13 @@ acvm = { version = "0.9.0", features = ["bn254"] }
 sled = "0.34.6"
 blake2 = "0.9.1"
 dirs = { version = "3.0", optional = true }
-downloader = { version = "0.2.7", optional = true, default-features = false, features = ["rustls-tls"] }
-indicatif = { version = "0.15.0", optional = true }
+reqwest = { version = "0.11.3", optional = true, default-features = false, features = ["stream", "rustls-tls"] }
+tokio = { version = "1", optional = true, features=["full"] }
```

Comment:
We likely don't need all the features on here. I'd like to find the minimal set.